vignettes/tech-dm-join.Rmd

```diff
@@ -26,7 +26,7 @@ Read more about relational data models in the vignette ["Introduction to Relatio
 
 First, we load the packages that we need:
 
-```{r message=FALSE, warning=FALSE}
```

Comment:
To be consistent with all the other vignettes, this is taken care of in `setup.R`.